### PR TITLE
Sync OpenStudio-HEScore and hescore-hpxml

### PR DESCRIPTION
**Pull request by nmerket**
## Pull Request Description

Pulling in any straggler changes from OpenStudio-HEScore

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] Code changes (must work)
- [ ] Test exercising your feature or bug fix. Check the coverage report in the build artifacts.
- [ ] All other unit tests passing
- [ ] Update translation docs

### CODE REVIEW

**Comment by bpark1327** [comment]
@nmerket I think it's ready for your review. I put some comments on the parts that need your attention.

File: hescorehpxml/base.py [hunk 1]
```diff
@@ -2415,8 +2428,6 @@ def get_systems_dhw(self, b):
                         'Tankless water heater efficiency cannot be estimated by shipment weighted method.')
                 else:
                     dhwyear = int(xpath(primarydhw, '(h:YearInstalled|h:ModelYear)[1]/text()', raise_err=True))
-                    if dhwyear < 1972:
-                        dhwyear = 1972
```
[the remark this answers is not in the document]
Comment (nmerket) [reply hescorehpxml/base.py:2419]: Yeah, go ahead and remove. That'll make it consistent with other years in the translation.

File: tests/test_translation.py [hunk 2]
```diff
@@ -1796,10 +1846,12 @@ def test_dhw_heat_pump_efficiency(self):
...
-        el.text = str(dt.datetime.today().year + 1)
-        self.assertRaisesRegex(InputOutOfBounds,
-                               'domestic_hot_water_year is out of bounds',
-                               tr.hpxml_to_hescore)
+        el.text = '1971'
+        self.assertRaisesRegex(
+            jsonschema.exceptions.ValidationError,
+            r'1971 is less than the minimum of 1972',
+            tr.hpxml_to_hescore
+        )
```
[the remark this answers is not in the document]
Comment (nmerket) [reply tests/test_translation.py:1854]: Ok. That makes sense.

**Review by nmerket — COMMENTED** [review]
Looks good to me. I answered your questions below.

**Pull request merged**